seerep_srv/seerep_core_pb/src/core_pb_image.cpp

```diff
@@ -44,9 +44,11 @@ boost::uuids::uuid CorePbImage::addData(const seerep::pb::Image& img)
   seerep_core_msgs::DatasetIndexable dataForIndices = CorePbConversion::fromPb(img);
 
   seerep_core_msgs::camera_intrinsics_query camintrinsics_query;
-  camintrinsics_query.uuidCameraIntrinsics = boost::lexical_cast<boost::uuids::uuid>(img.uuid_camera_intrinsics());
   camintrinsics_query.uuidProject = boost::lexical_cast<boost::uuids::uuid>(img.header().uuid_project());
```

Comment:
I just saw that this lexical cast has the same problem. Can you address this also in this PR, please?